src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -202,7 +204,13 @@ def parse_performance_features(log_lines, strategies, arguments):
 
   # Initialize all strategy stats as disabled by default.
   for strategy_type in strategy.LIBFUZZER_STRATEGY_LIST:
-    stats[strategy_column_name(strategy_type.name)] = 0
+    if (strategy_type.name in
+        strategy.LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE_TYPE and
+        strategy.LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE_TYPE[strategy_type.name]
+        == str):
+      stats[strategy_column_name(strategy_type.name)] = 'None'
```

Comment:
@oliverchang is there any different whether we use `'None'` or an empty string, WDYT? I definitely don't want `None` (aka `NULL`) as that should indicate that strategy didn't existed at the time of a particular run

Reply:
Good point, empty string may be better here. 'None' shouldn't result in a NULL value, but empty ('') is still cleaner as the "default" value for a string.